src/style.css

```
@@ -1 +1,11 @@
-/* styles go here */
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.registerForm .fieldset {
+  margin-top: 20px;
+}
```

You can just use `.fieldset`
```suggestion
.fieldset {
  margin-top: 20px;
}
```